e2etests/txtar.txt

```diff
@@ -171,6 +171,19 @@ a -> b: hi {
   style.underline: true
 }
 
+-- icon-style --
+
+lambda : {
+  icon: https://icons.terrastruct.com/aws%2FCompute%2FAWS-Lambda.svg
+  icon.style.border-radius: 20
```

Comment:
can you add more to this test? e.g. try with 0, 5, 20, 999

https://d2lang.com/tour/style/#border-radius